ui/store/actions.js

```diff
@@ -1391,6 +1391,19 @@ export function removeCollectible(address, tokenID, dontShowLoadingIndicator) {
   };
 }
 
+export function checkAndUpdateCollectiblesOwnershipStatus() {
+  return async (dispatch) => {
+    try {
+      await promisifiedBackground.checkAndUpdateCollectiblesOwnershipStatus();
+    } catch (error) {
+      log.error(error);
+      dispatch(displayWarning(error.message));
+    } finally {
+      await forceUpdateMetamaskState(dispatch);
```

Comment:
I don't think this step is needed. Background state changes are automatically propagated to the UI. We only need to "force update" the MetaMask state when we need to perform additional steps _after_ the state update has occurred. Since this is the last step here, that doesn't seem to be the case here.

Reply:
removed.